src/js/player.js

```diff
@@ -3031,6 +3031,42 @@ class Player extends Component {
    *         A promise with a Picture-in-Picture window.
```

Comment:
Comments may need to be updated as well there. Does the `Player#enterpictureinpicture` fire?

Reply:
👍  It does now